packages/web/src/components/search/addons/Highlight.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectGlobal } from 'emotion';
+
+import { escapeRegExp } from '../../../utils';
+
+// eslint-disable-next-line no-unused-expressions
+injectGlobal` 
```

Comment:
@bietkul  Is it good to use `injectGlobal` , instead, we can give className to the `div` and use selector for applying css.